piet-web/src/lib.rs

```diff
@@ -236,10 +236,7 @@ impl<'a> RenderContext for WebRenderContext<'a> {
     }
 
     fn current_transform(&self) -> Affine {
-        // todo
-        // current_transform() and get_transform() currently not implemented:
-        // https://github.com/rustwasm/wasm-bindgen/blob/f8354b3a88de013845a304ea77d8b9b9286a0d7b/crates/web-sys/webidls/enabled/CanvasRenderingContext2D.webidl#L136
-        Affine::default()
+        matrix_to_affine(self.ctx.get_transform().unwrap())
```

Comment:
what are the conditions under which this might fail? I might prefer to log this and return an identity affine? Not sure if anyone else has any opinions..

Reply:
I don't have very strong feelings but am ok with the crash (it seems more likely to be noticed to me). From having looked at a couple more things like this (text measurement), failures here are extremely unlikely in browsers that can actually run wasm.

Reply:
My slight preference is to let if fail rather than having some weird behaviour. But I'm also okay with logging some error message and return an identity affine